tests/drivers/adc/adc_api/boards/96b_aerocore2.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) Benjamin Björnsson <benjamin.bjornsson@gmail.com>
+ */
+
+#include <zephyr/dt-bindings/adc/adc.h>
```

Comment:
Nitpick (not blocking): This include should already be done in stm32xxxx.dtsi files

Reply:
Tried my best to remove those not needed, include is missing for C0, F4, H5 and U5.

Reply:
I can take care of this, the problem exists for more than just the SoCs from ST named above.